aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -1165,7 +1167,8 @@ std::tuple<Tensor, Tensor, Tensor, Tensor, c10::SymInt, c10::SymInt> _efficient_
       std::move(seed_t),
       std::move(offset_t),
       max_seqlen_q,
-      max_seqlen_k);
+      // TODO: why isn't this being set in the kernel?
```

Comment:
can you maybe make in issue or add more detail on this comment and what we expect